Client/Frontend/Login/LoginInfoTableViewCell.swift

```diff
@@ -0,0 +1,211 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import SnapKit
+import Storage
+import Shared
+import BraveUI
+
+protocol LoginInfoTableViewCellDelegate: AnyObject {
+    /// TextField Related Actions/ Delegates
+    func textFieldDidEndEditing(_ cell: LoginInfoTableViewCell)
+    func shouldReturnAfterEditingTextField(_ cell: LoginInfoTableViewCell) -> Bool
+    /// Table Row Commands Related Actions (Copy - Open)
+    func canPerform(action: Selector, for cell: LoginInfoTableViewCell) -> Bool
+    func didSelectOpenAndFill(_ cell: LoginInfoTableViewCell)
+    func didSelectReveal(_ cell: LoginInfoTableViewCell) -> Bool
+}
+
+class LoginInfoTableViewCell: UITableViewCell, TableViewReusable {
+    
+    // MARK: UX
+    
+    public struct UX {
+        static let horizontalOffset: CGFloat = 14
+    }
+
+    private let labelContainer = UIView()
+    var descriptionTextField = UITextField()
+    var highlightedLabel = UILabel()
+    
+    var placeholder: String? {
+        get {
+            descriptionTextField.placeholder
+        }
+        set {
+            descriptionTextField.placeholder = newValue
+        }
+    }
+
+    var displayDescriptionAsPassword: Bool = false {
+        didSet {
+            descriptionTextField.isSecureTextEntry = displayDescriptionAsPassword
+        }
+    }
+
+    var isEditingFieldData: Bool = false {
+        didSet {
+            guard isEditingFieldData != oldValue else { return }
```

Comment:
convert to `if`

Reply:
Fix at https://github.com/brave/brave-ios/pull/4124/commits/a52671a6b3e1b495a7427f86d89b4548203c6804